examples/MLIRAffine/affine-max.mlir

```diff
@@ -0,0 +1,9 @@
+func @main()
```

Comment:
- Please change `func` to `func.func`, the `func` is no longer a valid op.
- Please use the following style:
```
@main() {
  ...
  
 }
```